src/Rules/Conditions/Condition.cs

```diff
@@ -74,6 +74,5 @@ public static Condition Create(DelegateCondition.MatchesDelegate d, string descr
 
         public static Condition True = Create(e => true, ConditionDescriptions.True);
         public static Condition False = Create(e => false, ConditionDescriptions.False);
-        public static Condition DebugBreak = Create(e => { System.Diagnostics.Debugger.Break(); return true; });
```

Comment:
I'm not sure I agree with removing this. It will never get called unintentionally because it's never used. It's only a convenience for debugging rules, which are otherwise difficult to set breakpoints on. On the other hand, if you object to keeping it, it's pretty easy to add back as needed for debugging.

Reply:
I made it debug only as a compromise--it'll never make it into the NuGet package, and it's still easily accessible for testing and debugging